### DISCUSSION TITLE
iphone的地圖也太好用了吧

### DISCUSSION THREAD

**Discussion by chu109502015** [1101-GS4719-C/Python-Community-GS4719-C#1590] [General]
現在Apple地圖上新增的環視功能，是利用高解析度的 3D 攝影技術提供互動式街景，並運用擴增實境的逐步步行路線可以幫助使用者在不熟悉的區域中移動。用戶只要舉起 iPhone、掃描周圍的建築物，「地圖」app 將以現實環境為背景，顯示容易遵循的方向。
這樣就不會迷路了，該換iphone15了吧
https://tw.news.yahoo.com/%E5%8F%B0%E7%81%A3%E4%B9%9F%E6%9C%89%E4%BA%86-%E8%98%8B%E6%9E%9C-%E5%9C%B0%E5%9C%96-%E6%96%B0%E5%A2%9E%E7%92%B0%E8%A6%96-%E8%87%AA%E7%84%B6%E8%AA%9E%E9%9F%B3%E6%8C%87%E5%BC%95%E6%96%B0%E5%8A%9F%E8%83%BD-015510353.html

**Comment by FakeWood** [comment]
不過雖然輸入同樣的地名，Apple地圖跟Google地圖導航至的可能是不同地點，三五好友一起出門時可能要小心注意這點，上次出去玩我朋友就是這樣不見的@@
